scripts/shtf.py

```diff
@@ -0,0 +1,629 @@
+import json
+import logging
+import os
+import time
+from collections import defaultdict
+from dataclasses import dataclass, field
+from datetime import datetime
+from pathlib import Path
+from typing import Callable, DefaultDict, Optional, TypedDict, cast
+
+import click
+import requests
+import toml
+from eth_typing import BlockNumber
+from typing_extensions import NotRequired
+from web3 import Web3
+from web3.constants import ADDRESS_ZERO
+from web3.middleware import geth_poa_middleware
+from web3.types import BlockData
+
+import beamer.contracts
+import beamer.events
+from beamer.config import _merge_dicts
+from beamer.events import ClaimMade, DepositWithdrawn, RequestCreated, RequestFilled
+from beamer.typing import ChainId
+from beamer.util import load_ERC20_abi, setup_logging
+
+logging.disable()
+
+
+class MessageTypes:
```

Comment:
This seems to me an Enum

Reply:
We discussed this, but want to also post here - just for the protocol. I think that I don't want to use Enum here as it might be the pythonic way, but I find it the moronic way :D :D :D 

If I make this class a Enum - then as far as I understand I can't directly access the value under `MessageTypes.REQUEST_EXPIRED`, but now have to do `MessageTypes.REQUEST_EXPIRED.value` and I don't think I earn anything by doing this?

Or the other possibility `def __repr__(self): return self.value` on the enum class in order for it to behave like a class... 

I think I can live with this less than ideal Class over here :)